mongodb/mongodb.go

```diff
@@ -33,7 +33,12 @@ func New(config ...Config) *Storage {
 	cfg := configDefault(config...)
 
 	// Create data source name
-	var dsn = "mongodb://"
+	var dsn = "mongodb"
+	if cfg.Atlas == true {
```

Comment:
We can do `if cfg.Atlas {`

Reply:
Ah, didn't know you could do that with Go. Fixed that.